ImperatorToCk3Tests/ImperatorWorldTests/Provinces/PopsTests.cpp

```diff
@@ -0,0 +1,38 @@
+#include "gtest/gtest.h"
+#include <sstream>
+
+#include "../ImperatorToCK3/Source/Imperator/Provinces/Pops.h"
+#include "../ImperatorToCK3/Source/Imperator/Provinces/Pop.h"
+
+TEST(ImperatorWorld_PopsTests, popsDefaultToEmpty)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	ImperatorWorld::Pops pops;
+	pops.loadPops(input);
+
+	ASSERT_TRUE(pops.getPops().empty());
+}
+
+TEST(ImperatorWorld_PopsTests, popsCanBeLoaded)
```

Comment:
You should have your input also have a representation of the none form. If that's meant to be excluded, instead have a test demonstrating it gets excluded.

Reply:
Added ✔️